### PR TITLE
Adjust composer suggest

### PR DESCRIPTION
**Pull request by dereuromark**
@ehsanmx 
Is that what you meant?

Please also fix up require-dev then

PS: Synchronization will now be 0.2 from today on. Is this compatible with both 0.1 and 0.2?

### CODE REVIEW

**Comment by dereuromark** [comment]
ping @ehsanmx

**Comment by ehsanmx** [comment]
@gerner-spryker can you please take care of this in your new release?

**Comment by gerner-spryker** [comment]
Isn't it enough @dereuromark if we release this fix with the next PR? I mean this is a suggest on a "beta" module.

**Comment by dereuromark** [comment]
You can just merge this now and dont release (dev-master is sufficient).

File: composer.json [hunk 1]
```diff
@@ -8,8 +8,11 @@
     "spryker/propel-orm": "^1.0.0",
     "spryker/zend": "^2.1.0"
   },
+  "require-dev" : {
+    "spryker/synchronization": "^0.1.0"
```
Comment (dereuromark) [inline composer.json]: || ^0.2.0 ?

Comment (dereuromark) [reply composer.json]: Or directly use 0.2.0 (better/safer)

File: composer.json [hunk 2]
```diff
@@ -8,8 +8,11 @@
     "spryker/propel-orm": "^1.0.0",
     "spryker/zend": "^2.1.0"
   },
+  "require-dev" : {
+    "spryker/synchronization": "^0.1.0 || ^0.2.0"
+  },
   "suggest": {
-    "spryker/synchronizationBehavior": "^0.1.0"
+    "spryker/synchronization": "*"
```
Comment (dereuromark) [inline composer.json:15]: This should be a text, why one would want to use it.

Comment (gerner-spryker) [reply composer.json:15]: I will fix that in CORE-2977

**Pull request merged**